core/Cargo.toml

```diff
@@ -25,7 +25,7 @@ pest_derive = "2.0"
 shellexpand = "2.1"
 blake3 = "1.3"
 async-trait = "0.1"
-hyper = { version = "0.14", features = ["full"] }
+hyper = { version = "0.14", features = ["full", "backports", "deprecated"] }
```

Comment:
why `deprecated`

Reply:
So we get the deprecation warnings. Basically this gives us some of the new API + deprecation warning for some of the things that are removed. Step 1 towards migrating to 1.x.x